dynawo/sources/Models/CPP/Components/ModelLoadRestorativeWithLimits/DYNModelLoadRestorativeWithLimits.cpp

```diff
@@ -73,6 +73,8 @@ namespace DYN {
   UiYNum_(0),
   IrYNum_(0),
   IiYNum_(0),
+  preSwitchOff1_(0),
+  preSwitchOff2_(0),
```

Comment:
This should be initialized similarly to zLocal (with false)

Reply:
implementation has changed to align to other cpp models, those attributes are not anymore present